src/NzbDrone.Core/Indexers/IndexerFactory.cs

```diff
@@ -62,12 +67,31 @@ public List<IIndexer> SearchEnabled(bool filterBlockedIndexers = true)
 
             if (filterBlockedIndexers)
             {
-                return FilterBlockedIndexers(enabledIndexers).ToList();
+                return FilterBlockedIndexers(SubstituteIndexerUrls(enabledIndexers)).ToList();
             }
 
             return enabledIndexers.ToList();
         }
 
+        private IEnumerable<IIndexer> SubstituteIndexerUrls(IEnumerable<IIndexer> indexers)
```

Comment:
Would we want to persist these changes to the DB instead of applying them when they are used?

Definitely pros and cons to both, but changing it might make for a more obvious fix to the user, unless the notification message will be displayed when editing (at some point?).